aten/src/ATen/native/TensorTransformations.h

```diff
@@ -10,8 +10,11 @@ namespace at {
 namespace native {
 
 static inline void flip_check_errors(int64_t total_dims, int64_t flip_dims_size, IntArrayRef dims) {
```

Comment:
flip_check_errors used to be called in from flip_cpu, too. I believe it was accidentally removed by https://github.com/pytorch/pytorch/pull/13344/files. Could we add it back to flip_cpu?

Reply:
Oh I see the above discussion suggests these checks may be redundant on CPU and we don't need them. Thanks @kshitij12345 for pointing out they're already tested for.